config/routes.rb

```diff
@@ -3,4 +3,5 @@
   ActiveAdmin.routes(self)
   devise_for :users
   root to: 'movies#index'
+  get 'aws_texts/index' => 'aws_texts#index'
```

Comment:
`resources` を使いましょう。

```suggestion
  resources 'aws_texts', only: [:index, :show]
```